java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -213,4 +217,104 @@ public interface BitmapBaseCommands {
      */
     CompletableFuture<Long> bitpos(
             String key, long bit, long start, long end, BitmapIndexType offsetType);
+
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified subcommands.
+     *
+     * @see <a href="https://redis.io/commands/bitfield/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The subcommands to be performed on the binary value of the string at <code>
+     *     key</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldOptions.BitFieldGet} gets the value in {@link BitFieldOptions.Offset}
+     *           or {@link BitFieldOptions.OffsetMultiplier} based on encoding being {@link
+     *           BitFieldOptions.SignedEncoding} or {@link BitFieldOptions.UnsignedEncoding}.
+     *       <li>{@link BitFieldOptions.BitFieldSet} sets the value in {@link BitFieldOptions.Offset}
+     *           or {@link BitFieldOptions.OffsetMultiplier} based on encoding being {@link
+     *           BitFieldOptions.SignedEncoding} or {@link BitFieldOptions.UnsignedEncoding}.
+     *       <li>{@link BitFieldOptions.BitFieldIncrby} increases or decreases the value in {@link
+     *           BitFieldOptions.Offset} or {@link BitFieldOptions.OffsetMultiplier} based on encoding
+     *           being {@link BitFieldOptions.SignedEncoding} or {@link
+     *           BitFieldOptions.UnsignedEncoding}.
+     *       <li>{@link BitFieldOptions.BitFieldOverflow} determines behaviour of {@link
+     *           BitFieldOptions.BitFieldSet} or {@link BitFieldOptions.BitFieldIncrby} when these
+     *           operations result in under or overflows.
+     *     </ul>
+     *     <br>
+     *     <ul>
+     *       <li>{@link BitFieldOptions.Offset} and {@link BitFieldOptions.OffsetMultiplier} must be
+     *           greater than or equal to 0.
+     *       <li>{@link BitFieldOptions.SignedEncoding} must be less than 64.
+     *       <li>{@link BitFieldOptions.UnsignedEncoding} must be less than 65.
+     *     </ul>
+     *
+     * @return An array of results from subcommands <code>GET</code>, <code>SET</code>, or <code>
+     *     INCRBY</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldOptions.BitFieldGet} returns the value in <code>
```

Comment:
As an alternative option you say that command returns
> An array of subcommands results, which is defined/described for each subcommand.

I don't like mine wording, but I propose to refer to existing subcommand docs instead of copying them here.

Reply:
And wrap array with `code`

Reply:
Sorry what do you mean by this comment?

Reply:
> An <code>array</code> ...